src/saveload/oldloader_sl.cpp

```diff
@@ -465,10 +465,10 @@ static void FixTTOCompanies()
 	}
 }
 
-static inline byte RemapTTOColour(byte tto)
+static Colours RemapTTOColour(Colours tto)
```

Comment:
Lost its `inline` - intentional?

Reply:
Hmm, probably but not relevant to this change I think.